checker/src/main/java/org/checkerframework/checker/signedness/qual/SignednessBottom.java

```diff
@@ -17,7 +17,7 @@
  * @checker_framework.manual #signedness-checker Signedness Checker
  * @checker_framework.manual #bottom-type the bottom type
  */
-@SubtypeOf({SignednessGlb.class})
+@SubtypeOf({SignednessGlb.class, SignedPositive.class})
```

Comment:
`SignednessPosstive` should not be here since it's an alias.

Reply:
Thanks for catching that.